blocks/featured/index.js

```diff
@@ -93,9 +23,95 @@ export default registerBlockType(
 			__( 'Travel' )
 		],
 
-		edit() {
```

Comment:
This [is outside](https://github.com/xwp/travel/pull/34/files#diff-f13e039fde62e426638518f5cefb1783L87) of this PR's diff. But could the title of this block be 'Featured Destinations,' instead of 'Featured'? Or something that's descriptive.
```javascript
        title: __( 'Featured Destinations' ),
 		category: 'common',
 		icon: 'wordpress-alt',
```

Reply:
Thanks for making this change.